backend/api/views/pm.py

```diff
@@ -34,34 +34,19 @@ def get_pm_by_email(email):
     )
 
 
-@pm.route("/portfolio_manager/all_fps/<id>", methods=["GET"])
```

Comment:
List of Field Partners removed from Portfolio Manager table - see model file for elaboration.